index.js

```diff
@@ -78,6 +109,34 @@ class TrezorKeyring extends EventEmitter {
     return this.model;
   }
 
+  /**
+   * Gets the vendor, if known.
+   *
+   * @returns {"trezor" | "onekey"}
+   */
+  async getVendor() {
+    return this.vendor || (this.vendor = await this.fetchVendor());
+  }
+
+  /**
+   * fetch vendor by call getFeatures
+   * @param {object} features
+   * @returns {'onekey' | 'trezor' | null}
+   */
+  fetchVendor() {
```

Comment:
Does this need to be public if you can call this through `getVendor`? If not, what do you think about prepending this with `_`?

Reply:
Nice suggesstion. Added.